rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Both Getters and Setters for `@book` and `@person` instance variables should be present as they should be modified outside the class so I'd advise you to place the code at line `3` in the `attr_accessor` right above.